index.html

```diff
@@ -30,7 +29,7 @@
           />
         </a>
 
-        <div>
+        <nav class="nav">
           <ul class="navigation__list">
             <li class="navigation__item">
               <a class="navigation__link" href="#">Home</a>
```

Comment:
These links do not become orange on hover, can you try to make that happen? thanks

Reply:
alright i will do it. Thanks for your feedback!